src/main/java/Flash.java

```diff
@@ -0,0 +1,138 @@
+import java.io.File;
+import java.sql.Array;
+import java.util.Scanner;
+import java.util.List;
+import java.util.ArrayList;
+
+public class Flash {
+
+    private static final List<Task> tasks = new ArrayList<>();
+
+    public static void displayTasks() {
+        System.out.println("____________________________________________________________");
+        for(int i = 0; i < tasks.size(); i++) {
+            System.out.println(i+1 + "." + tasks.get(i));
+        }
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void markTask(String input) throws FlashException {
```

Comment:
The logic for marking and unmarking tasks is almost identical. You can refactor this into a single method to avoid duplication.